pkg/connector/ble/ble.go

```diff
@@ -209,14 +282,21 @@ func NewConnection(ctx context.Context, vin string) (*Connection, error) {
 	return NewConnectionToBleTarget(ctx, vin, nil)
 }
 
-func NewConnectionToBleTarget(ctx context.Context, vin string, target Advertisement) (*Connection, error) {
+// NewConnectionToBleTarget creates a new BLE connection to the given target.
```

Comment:
Go style conventions say "BLE" is implied by the package name and initialisms should be uppercased. I'd call this `NewConnectionFromScanResult`.